### PR TITLE
Fixed #25667 -- Fixed admindocs docutils initial_header_level.

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

**Comment by timgraham** [comment]
It makes sense, but I think we should try to add a test in `tests/admin_docs/tests.py`. Would you like to try?

File: tests/admin_docs/tests.py [hunk 1]
```diff
@@ -390,6 +390,8 @@ def test_model_docstring_renders_correctly(self):
             'reference external" href="/admindocs/models/myapp.company/">'
             'myapp.Company</a> where the person works.</dd></dl>'
         )
+        descriptionheading = '<h3>Description</h3>'
```
Comment (timgraham) [inline tests/admin_docs/tests.py]: I think I would put the test in `test_description_output` instead -- did you consider that?

Comment (scop) [inline tests/admin_docs/tests.py]: No, but maybe it'd be better placed there indeed as this stuff isn't actually about models (or descriptions for that matter). Looks like moving it to where you suggest would require changes in several TestUtils.test_*() methods though.

Comment (timgraham) [inline tests/admin_docs/tests.py]: I think you could add a separate test that uses `utils.parse_rst()` directly then.

**Comment by timgraham** [comment]
merged in 4d0f8831a7498ab3b1ebcf4cafa2ee234503e4f8, thanks!

**Pull request closed without merging**